source/extensions/filters/http/lua/lua_filter.cc

```diff
@@ -689,6 +689,8 @@ void Filter::scriptLog(spdlog::level::level_enum level, const char* message) {
     return;
   case spdlog::level::off:
     NOT_IMPLEMENTED_GCOVR_EXCL_LINE;
+  default:
+    NOT_REACHED_GCOVR_EXCL_LINE;
```

Comment:
This seems concerning. Is there a new level that we need to handle here?

/wait

Reply:
There is no new level to handle. The new value is `n_levels` which is a number of levels, used internally in spdlog.

https://github.com/gabime/spdlog/blob/v1.x/include/spdlog/common.h#L148

Reply:
Should I handle it explicitly (instead of using the `default` case) with a comment?

Reply:
I think it would be better. Otherwise we may actually miss a new level, even though it is highly unlikely.

Reply:
+1

Reply:
Done.